src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/dto/ChangelogValuesDTO.kt

```diff
@@ -0,0 +1,28 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.dto
+
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.Changelog
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.service.UserDetail
+import java.time.OffsetDateTime
+import java.util.UUID
+
+data class ChangelogValuesDTO(
+  val changelogId: UUID,
+  val referralId: UUID,
+  val topic: String,
+  val changedAt: OffsetDateTime,
+  val name: String,
```

Comment:
@JakeGoodwin22 , what is the purpose of this name and I am seeing you are not sending back the changedBy back to client. I am checking the designs, we need that value to fill in change_made_by

Reply:
Changed as discussed